### ISSUE TITLE
Disable eMIS prefill 

### ISSUE THREAD

**Issue by LindseySaari** [department-of-veterans-affairs/va.gov-team#61658]
## Issue Description
As part of the eMIS deprecation, we should disable eMIS prefill for military information. All of the in progress forms utilize the eMIS prefill method for military information, but on September 1st eMIS will be deprecated in favor of VaProfile. 

As part of the deprecation, we should disable military_information that comes from eMIS. See the [prioritization doc](https://vfs.atlassian.net/wiki/spaces/BTT/pages/2274426885/EMIS+deprecation+prioritization) and the [EMIS pre-fill impact](https://vfs.atlassian.net/wiki/spaces/BTT/pages/2286714883/EMIS+Pre-fill+Impact+WIP) documentation. The initialize_military_information method is used to obtain military related information from eMIS. 

TLDR; Disabling pre-fill won't necessarily be a breaking change, but rather degrade the veteran experience related to form pre-fill by requiring more typing for the veteran

---
## Tasks
- [ ] Disabling [eMIS pre-fill for military info](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/models/form_profile.rb#L231-L232)

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)
